.travis.yml

```
@@ -81,7 +81,7 @@ install:
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
   # Install libraries that require specific development headers, but not during lint test
-  - if ! [ "$LINT_CHECK" = "1" ]; then pip install "pymssql<3.0" MySQL-python pyodbc; fi
+  - if ! [ "$LINT_CHECK" = "1" ]; then pip install pymssql==2.1.5 MySQL-python pyodbc; fi
```

Better not to use fixed versions because it is a big constraint on other packages
```suggestion
  - if ! [ "$LINT_CHECK" = "1" ]; then pip install "pymssql<=2.1.5" MySQL-python pyodbc; fi
```